docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -177,9 +177,9 @@ The Solutionism Trap
 
 This trap occurs when we assume that the best solution to a problem
 may involve technology, and fail to recognize other possible solutions outside of
-this realm.  Solutionist approaches may also not be appropriate in situations where definitions of fairness are particularly
-contested and changed and models become obsolete. See the "Construct
-Validity" section in Fairlearn's user guide.
+this realm.  Solutionist approaches may also not be appropriate in situations
+where definitions of fairness may change over time (see the Formalism Trap below for more).
+See also the "Construct Validity" section in Fairlearn's user guide.
```

Comment:
For linking to other sections (and traps) we can first tag those sections
```
.. _<tag>:
<section header>
<section>

```
and then reference it
```
:ref:`<tag>`
```
I'm sure you'll find plenty of examples in the docs (including this very file). 🙂